scripts/pi-hole/js/footer.js

```diff
@@ -85,6 +85,13 @@ $("#pihole-disable-5m").on("click", function(e){
     piholeChange("disable","300");
     setTimeout(function(){piholeChanged("enabled");},300000);
 });
+$("#pihole-disable-custom").on("click", function(e){
+    e.preventDefault();
+    var custVal = $("#customTimeout").val();
+    custVal = $("#btnMins").hasClass("active") ? custVal * 60 : custVal;
+    piholeChange("disable",custVal);
+    setTimeout(function(){piholeChanged("enabled");},300000);
```

Comment:
This does not make sense - it will change the status display back to enabled after 5 minutes (300 seconds) whatever you specify for the interval